### PR TITLE
topology: cache expiry

### PR DESCRIPTION
**Pull request by mcutalo88**
### Description
Expire topology cache that is older than 2 hours.

### Testing Performed
Tested locally by reducing the expiry time to 60 seconds.

### GitHub Issue

This PR makes progress on the Topology API #30 issue.

### CODE REVIEW

File: backend/clutch-config.yaml [hunk 1]
```diff
@@ -24,6 +24,16 @@ modules:
   - name: clutch.module.k8s
   - name: clutch.module.kinesis
 services:
+  - name: clutch.service.db.postgres
```
Comment (dschaller) [inline backend/clutch-config.yaml:27]: Should we keep this in the default config?

Comment (mcutalo88) [reply backend/clutch-config.yaml:27]: ah yeah sorry I usually comment on this, but ill remove the config changes once the review is complete.

File: backend/service/topology/cache.go [hunk 2]
```diff
@@ -172,3 +173,32 @@ func (c *client) deleteCache(ctx context.Context, id string) error {
 	c.scope.SubScope("cache").Counter("delete.success").Inc(1)
 	return nil
 }
+
+func (c *client) expireCache(ctx context.Context) {
+	// Delete all entries that are older than two hours
+	const expireQuery = `
+		DELETE FROM topology_cache WHERE updated_at <= NOW() - INTERVAL '120m';
```
Comment (danielhochman) [inline backend/service/topology/cache.go:180]: nit: i'm sure this is minutes but the docs are terrible in trying to verify that. write out minutes in full please.

File: backend/service/topology/cache.go [hunk 3]
```diff
@@ -172,3 +173,32 @@ func (c *client) deleteCache(ctx context.Context, id string) error {
...
+	`
+
+	ticker := time.NewTicker(time.Minute * 20)
+
+	go func() {
+		<-ctx.Done()
+		ticker.Stop()
+	}()
+
+	for ; true; <-ticker.C {
```
Comment (danielhochman) [inline backend/service/topology/cache.go:190]: see https://github.com/lyft/clutch/pull/556#discussion_r508556187

**Review by danielhochman — COMMENTED** [review]
also let's add stats

**Comment by mcutalo88** [comment]
Need to close and open the CLA check is stuck

**Review by rithujohn191 — APPROVED** [review]
LGTM

**Pull request merged**